Cargo.toml

```diff
@@ -10,18 +10,18 @@ keywords = ["filecoin", "web3", "wasm"]
 exclude = ["examples", ".github"]
 
 [target.'cfg(target_arch = "wasm32")'.dependencies]
-fil_actor_account = { version = "7.0.2", path = "./actors/account" }
-fil_actor_verifreg = { version = "7.0.2", path = "./actors/verifreg" }
-fil_actor_cron = { version = "7.0.2", path = "./actors/cron" }
-fil_actor_market = { version = "7.0.2", path = "./actors/market" }
-fil_actor_multisig = { version = "7.0.2", path = "./actors/multisig" }
-fil_actor_paych = { version = "7.0.2", path = "./actors/paych" }
-fil_actor_power = { version = "7.0.2", path = "./actors/power" }
-fil_actor_miner = { version = "7.0.2", path = "./actors/miner" }
-fil_actor_reward = { version = "7.0.2", path = "./actors/reward" }
-fil_actor_system = { version = "7.0.2", path = "./actors/system" }
-fil_actor_init = { version = "7.0.2", path = "./actors/init" }
-fil_actors_runtime = { version = "7.0.2", path = "./actors/runtime" }
+fil_actor_account = { version = "8.0.0-alpha.1", path = "./actors/account" }
+fil_actor_verifreg = { version = "8.0.0-alpha.1", path = "./actors/verifreg" }
+fil_actor_cron = { version = "8.0.0-alpha.1", path = "./actors/cron" }
+fil_actor_market = { version = "8.0.0-alpha.1", path = "./actors/market" }
+fil_actor_multisig = { version = "8.0.0-alpha.1", path = "./actors/multisig" }
+fil_actor_paych = { version = "8.0.0-alpha.1", path = "./actors/paych" }
+fil_actor_power = { version = "8.0.0-alpha.1", path = "./actors/power" }
+fil_actor_miner = { version = "8.0.0-alpha.1", path = "./actors/miner" }
+fil_actor_reward = { version = "8.0.0-alpha.1", path = "./actors/reward" }
+fil_actor_system = { version = "8.0.0-alpha.1", path = "./actors/system" }
+fil_actor_init = { version = "8.0.0-alpha.1", path = "./actors/init" }
+fil_actors_runtime = { version = "8.0.0-alpha.1", path = "./actors/runtime" }
```

Comment:
Unfortunately, `cargo set-version` (from the `cargo-edit` crate) does not discover these versions in the manifest, and therefore does not upgrade them. Had to do manually.

Still need to find the right release tool.

`cargo release` seemed great, but it hangs when publishing. Bug report: https://github.com/crate-ci/cargo-release/issues/429.